RFC-0010-mixed-case-identifier-support.md

```diff
@@ -0,0 +1,201 @@
+# **RFC-0010 for Presto**
+
+## Mixed case identifiers
+
+Proposers
+
+* Reetika Agrawal
+
+## Summary
+
+Improve Presto's identifier (schema, table & column names) handling to align with SQL standards, ensuring better interoperability with case-sensitive
+and case-normalizing databases while minimizing SPI-breaking changes.
+
+## Background
+
+Presto treats all identifiers as case-insensitive, normalizing them (typically to lowercase). This creates issues when
+querying databases that are case-sensitive (e.g., MySQL, PostgreSQL) or case-normalizing to uppercase (e.g., Oracle,
+DB2). Without a standard approach, identifiers might not match the actual names in the underlying data sources, leading
+to unexpected query failures or incorrect results. Additionally, inconsistent handling of delimited and non-delimited
+identifiers across different connectors further complicates cross-engine compatibility.
```

Comment:
Can you elaborate more on `inconsistent handling of delimited and non-delimited
identifiers`